src/main/java/org/ngafid/events/Event.java

```diff
@@ -35,6 +35,10 @@ public class Event {
 
     private Integer otherFlightId = null;
 
+    private String systemId;
+
+    private String tail;
+
     private RateOfClosure rateOfClosure;
 
     public Event(String startTime, String endTime, int startLine, int endLine, double severity) {
```

Comment:
Is this constructor still used anywhere else? Would using this constructor break something if there is no system ID or tail number provided?

Reply:
Yeah, this constructor was used on other files and I had to make a new one just to accomodate for adding the system and tail id.